### PR TITLE
feat: make a lazy load directive

### PR DESCRIPTION
**Pull request by williamareynolds**
Lazily loads a target element based on a trigger elements
visibility in the viewport.

Currently a bit of a work in progress, but we need it ASAP.

### CODE REVIEW

File: projects/flosportsinc/ng-lazy-load/src/lib/flo-lazy-load.module.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {CommonModule} from '@angular/common';
```
Comment (bexux) [inline projects/flosportsinc/ng-lazy-load/src/lib/flo-lazy-load.module.ts]: you can remove these semicolons

Comment (williamareynolds) [reply projects/flosportsinc/ng-lazy-load/src/lib/flo-lazy-load.module.ts]: I just removed all the semicolons. Angular generated them, so there were several in other files breaking linting as well.

Comment (bexux) [reply projects/flosportsinc/ng-lazy-load/src/lib/flo-lazy-load.module.ts]: blaming it on angular, smh

**Review by bexux — APPROVED** [review]

**Pull request merged**